developer-docs-site/docs/tutorials/your-first-nft-sdk.md

```diff
@@ -0,0 +1,376 @@
+
+## Step 1: Pick an SDK
+
+* [Official Aptos Python SDK][python-sdk]
+* Official Aptos Typescript SDK -- TBA
+* Official Aptos Rust SDK -- TBA
+
+## Step 2: Run the Example
+
+Each SDK provides an examples directory. This tutorial covers the `simple-nft` example.
+
+<Tabs>
+  <TabItem value="python" label="Python">
+
+In the SDK directory run: `python -m examples.simple-nft`
+  </TabItem>
+</Tabs>
+
+## Step 3: Understand the Output
+
+The following output should appear after executing the `simple-nft` example, though some values will be different:
+
+```
+=== Addresses ===
+Alice: 0x9df0f527f3a0b445e4d5c320cfa269cdefafc7cd1ed17ffce4b3fd485b17aafb
+Bob: 0xfcc74af84dde26b0050dce35d6b3d11c60f5c8c58728ca3a0b11035942a0b1de
+
+=== Initial Balances ===
+Alice: 20000
+Bob: 20000
+
+=== Creating Collection and Token ===
+Alice's collection: {
+    "description": "Alice's simple collection",
+    "maximum": "18446744073709551615",
+    "mutability_config": {
+        "description": false,
+        "maximum": false,
+        "uri": false
+    },
+    "name": "Alice's",
+    "supply": "1",
+    "uri": "https://aptos.dev"
+}
+Alice's token balance: 1
+Alice's token data: {
+    "default_properties": {
+        "map": {
+            "data": []
+        }
+    },
+    "description": "Alice's simple token",
+    "largest_property_version": "0",
+    "maximum": "1",
+    "mutability_config": {
+        "description": false,
+        "maximum": false,
+        "properties": false,
+        "royalty": false,
+        "uri": false
+    },
+    "name": "Alice's first token",
+    "royalty": {
+        "payee_address": "0x9df0f527f3a0b445e4d5c320cfa269cdefafc7cd1ed17ffce4b3fd485b17aafb",
+        "royalty_points_denominator": "1000000",
+        "royalty_points_numerator": "0"
+    },
+    "supply": "1",
+    "uri": "https://aptos.dev/img/nyan.jpeg"
+}
+
+=== Transferring the token to Bob ===
+Alice's token balance: 0
+Bob's token balance: 1
+
+=== Transferring the token back to Alice using MultiAgent ===
+Alice's token balance: 1
+Bob's token balance: 0
+```
+
+This example demonstrates:
+
+* Initializing the REST and Faucet clients
+* The creation of two accounts: Alice and Bob
+* The funding and creation of Alice and Bob's accounts
+* The creation of a collection and a token using Alice's account
+* Alice offering a token and Bob claiming it
+* Bob unilaterally sending the token to Alice via a multiagent transaction
+
+## Step 4: The SDK in Depth
+
+### Step 4.1: Initializing the Clients
+
+In the first step, the example initializes both the REST and Faucet clients. The REST client interacts with the REST API, whereas the Faucet client is a devnet
+ service for creating and funding accounts.
```

Comment:
`whereas the Faucet client interacts with the Faucet devnet service for creating and funding accounts`